src/css/style.css

```diff
@@ -688,3 +688,118 @@ textarea {
     background-position: top right;
   }
 }
+
+/* Modal styles */
+.modal-container {
+  display: none;
+}
+
+.modal {
+  position: fixed;
+  background-color: rgba(128, 128, 128, 0.9);
+  inset: 0;
+  z-index: 10000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.modal-box {
+  max-width: min(700px, 90%);
+  background-color: var(--color-white);
+  border-radius: 8px;
+  padding: var(--size-300);
+  overflow-y: auto;
+}
+
+.modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+}
+
+.modal-title {
+  font-family: Poppins, sans-serif;
+  font-size: 30px;
+  font-weight: 700;
+  text-align: left;
+}
+
+.timeline {
+  display: flex;
+  gap: 10px;
+  font-size: 12px;
+  color: var(--color-nav-p);
+  margin-bottom: 20px;
+}
+
+.modal .timeline li {
```

Comment:
- [ ] Kindly align the image in the mobile view to match that of the image in the provided design

Your design
![image](https://user-images.githubusercontent.com/81932478/188738262-e85855cd-2a15-4e0a-b47d-b0476397efe0.png)

Provided design
![image](https://user-images.githubusercontent.com/81932478/188738341-c338a1f9-2b5d-4d54-a93e-c959ac5f1c95.png)